osu.Android/GameplayScreenRotationLocker.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Android.Content.PM;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Game;
+
+namespace osu.Android
+{
+    public class GameplayScreenRotationLocker : Component
+    {
+        private Bindable<bool> localUserPlaying;
+
+        [BackgroundDependencyLoader]
+        private void load(OsuGame game)
+        {
+            localUserPlaying = game.LocalUserPlaying.GetBoundCopy();
+            localUserPlaying.BindValueChanged(userPlaying => updateLock(userPlaying));
```

Comment:
Change to `localUserPlaying.ValueChanged += updateLock;`

Reply:
Method group would also work. I'd slightly prefer that to an event subscription actually but unsure if it matters.

Reply:
`BindValueChanged` exists mostly to allow passing `true` as the second argument to run it immediately.
The first line is literally `ValueChanged += onChange;`

Reply:
Actually, I went with `BindValueChanged` as I thought we might want to run callback immediately in this scenario but I'm not sure it is actually necessary anymore.